SECURITY.md

```diff
@@ -62,7 +62,7 @@ In Maven, you can use the [os-maven-plugin](https://github.com/trustin/os-maven-
     <dependency>
       <groupId>io.netty</groupId>
       <artifactId>netty-tcnative-boringssl-static</artifactId>
-      <version>1.1.33.Fork26</version>
+      <version>2.0.3.Final</version>
```

Comment:
Considering that there is a known bug, and we can't tell people to just use the latest version (1.5 isn't out yet), it seems reasonable to point to the latest version.  (for 1.3 and earlier it doesn't matter so much, we just tell people to upgrade)